### PR TITLE
Use messages from language labels in JS config (fixes #76)

### LINKED ISSUE #76: Übersetzung innerhalb von Dialogboxen in English auf deutschen Seiten
- Contao: 4.9.11
- Extension: 3.1.5

Im Frontend werden die Fehlermeldung innerhalb der Overlays immer in Englisch ausgeben.

screen-20210127_SBiLWHj7

### PR DESCRIPTION
**Pull request by gmpf**
`ConfigGenerator#generateJavaScriptConfig()` should provide the message labels from the language files. That config then has to be passed in `handler.js`.

### CODE REVIEW

**Comment by gmpf** [comment]
By the way, `ConfigGenerator#generateLabels()` provides even more labels than just messages. All are set in `ConfigGenerator#generateFromWidgetAttributes()`, but not used anywhere.

**Review by fritzmg — APPROVED** [review]

File: src/ConfigGenerator.php [hunk 1]
```diff
@@ -74,6 +75,7 @@ public function generateJavaScriptConfig(UploaderConfig $config)
             'minSizeLimit' => $config->getMinSizeLimit(),
             'sizeLimit' => $config->getMaxSizeLimit(),
             'uploadButtonTitle' => $config->getUploadButtonTitle(),
+            'messages' => $labels['messages'],
```
Comment (qzminski) [inline src/ConfigGenerator.php:78]: I think we can safely use `$config->getLabels()['messages']` here instead of having another variable.

**Review by qzminski — CHANGES_REQUESTED** [review]

File: src/ConfigGenerator.php (hunk 1 above)
Comment (gmpf) [reply src/ConfigGenerator.php]: Sure can. :) Just to clarify why I used a variable there: If you would have wanted to add more stuff from `ConfigGenerator#generateLabels()` later on, the next diff would have been clearer that way (kind of the same reasoning for the already-existing code using a trailing comma after the final element). But that was purely speculative on my part. Changed.

**Review by qzminski — APPROVED** [review]

**Comment by aschempp** [comment]
Thank you @gmpf

**Pull request merged**